packages/app/components/feed/feed.md.tsx

```diff
@@ -264,18 +264,18 @@ const SuggestedUsers = () => {
           Get the app
         </Text>
         <View tw="flex flex-row items-center justify-between p-4">
-          <Link
-            tw="font-space-bold dark:text-white"
+          <TextLink
```

Comment:
if only has single text, we should use the `TextLink` component to avoid a warning from `next.js`.